tests/Unit/CssInlinerTest.php

```diff
@@ -3651,12 +3650,11 @@ public function inlineCssNotCopiesInlinableRuleAfterImportRuleToStyleElement():
     }
 
     /**
-     * @return array<string, array<int, string>>
+     * @return DataProvider<string, array<int, string>>
```

Comment:
As we're changing the type anyway, can the nested type be changed to specifiy a `list` of exactly three strings?

Reply:
Done and repushed.